.github/workflows/gradle.yml

```
@@ -0,0 +1,65 @@
+name: CD Workflow
+on:
+  push:
+    branches: [ "develop" ]
+  workflow_dispatch:
+
+jobs:
+  docker:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+
+    steps:
+      # JDK setting - github actions에서 사용할 JDK 설정
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
```

```suggestion
      - name: Checkout Repository
         uses: actions/checkout@v3
         
      - name: Set up JDK 17
        uses: actions/setup-java@v3
```

- name (job description)
uses (plugins)

위와 같은 컨벤션을 맞춰 헷갈리지 않게 checkout 에도 이름을 붙이는게 좋을거 같아요

---

확인했어요